### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Serzh108**

### CODE REVIEW

File: pages/styles.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: calc(100vw - 420px); 
+  height: fit-content;;
+  padding: 92px 114px 92px 135px;
+  background: #17191D;
+`
+
+export const ContentText = styled.p`
```
Comment (zAndrey95) [inline pages/styles.ts:43]: similar to AdvertText

File: pages/styles.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+export const BlockWrapper = styled.div`
+  display: flex;
```
Comment (zAndrey95) [inline pages/styles.ts:4]: I suggest this block (4,5,6 lines) create few universal styles and using

Comment (Serzh108) [reply pages/styles.ts:4]: Done

File: pages/styles.ts (hunk 1 above)
Comment (Serzh108) [reply pages/styles.ts:43]: I corrected this in my current work (other branch)

**Pull request merged**